sentinelsat/sentinel.py

```diff
@@ -6,7 +6,7 @@
 import re
 import shutil
 import warnings
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElemTree
```

Comment:
Hmm. I'd say you might as well `from xml.etree import ElementTree`.

Reply:
Actually, even in the [Python docs](https://docs.python.org/3/library/xml.etree.elementtree.html) they use `ET`.

Reply:
Agreed